provider/cmd/pulumi-resource-purrl/schema.json

```diff
@@ -31,18 +33,21 @@
         "nodejs": {
             "dependencies": {
                 "@pulumi/pulumi": "^3.0.0"
-            }
+            },
+            "packageName": "@pulumiverse/pulumi-purrl"
```

Comment:
In general for Pulumiverse, drop the `pulumi-` part from the package name, so `@pulumiverse/purrl`.

Reply:
done!